queries.sql

```diff
@@ -10,3 +10,45 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE NOT name = 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN  10.4 AND 17.3;
+
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+
+BEGIN;
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+SELECT * FROM animals;
+COMMIT;
+
+BEGIN;
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS null;
+SELECT * FROM animals;
+COMMIT;
+
+BEGIN;
+DELETE FROM animals;
+SELECT * FROM animals;
+ROLLBACK;
```

Comment:
- [ ] Kindly add this line at the end of your query 👇🏽 

``SELECT COUNT(*) FROM ANIMALS;``

It is to verify if all records in the animals table still exists as required 👇🏽 

> After the rollback verify if all records in the animals table still exists.